meson.build

```diff
@@ -75,6 +75,7 @@ gtkmm = dependency('gtkmm-3.0', version : ['>=3.22.0'])
 dbusmenu_gtk = dependency('dbusmenu-gtk3-0.4', required: get_option('dbusmenu-gtk'))
 giounix = dependency('gio-unix-2.0', required: get_option('dbusmenu-gtk'))
 jsoncpp = dependency('jsoncpp')
+pugixml = dependency('pugixml')
```

Comment:
I suppose a wrap file could be appreciated. But on the other hand, pugixml is packaged in most distros, so I don't know if this is really necessary.

Reply:
If a wrap file is available, i'll go for it 😄 
Otherwise think about adding the dependency in the readme.

Reply:
The wrap file doesn't exist in the wrapdb. I updated the README.